<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnJugador</class>
 <widget class="QWidget" name="UnJugador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="boton_pedir_carta">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pedir Carta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_retirarse">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Retirarse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_doblar">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Doblar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_dividir">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dividir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>660</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldo:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pantalla">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="selector_apuesta">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Introduce una apuesta inicial: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_empezar">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Empezar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_retirada">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>245</y>
     <width>500</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mensaje de fin de juego</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>650</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valor de la mano:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pantalla_2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>660</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pantalla_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_valor_banca">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valor de la banca:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_salir">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_reiniciar">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>80</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_continuar">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>150</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mesa1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>740</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta1J">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta2J">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta3J">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta4J">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta5J">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta1B">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta2B">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta3B">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta4B">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Carta5B">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>71</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Mesa1</zorder>
  <zorder>boton_pedir_carta</zorder>
  <zorder>boton_retirarse</zorder>
  <zorder>boton_doblar</zorder>
  <zorder>boton_dividir</zorder>
  <zorder>label</zorder>
  <zorder>pantalla</zorder>
  <zorder>selector_apuesta</zorder>
  <zorder>label_2</zorder>
  <zorder>boton_empezar</zorder>
  <zorder>label_retirada</zorder>
  <zorder>label_3</zorder>
  <zorder>pantalla_2</zorder>
  <zorder>pantalla_3</zorder>
  <zorder>label_valor_banca</zorder>
  <zorder>boton_salir</zorder>
  <zorder>boton_reiniciar</zorder>
  <zorder>boton_continuar</zorder>
  <zorder>Carta1J</zorder>
  <zorder>Carta2J</zorder>
  <zorder>Carta3J</zorder>
  <zorder>Carta4J</zorder>
  <zorder>Carta5J</zorder>
  <zorder>Carta1B</zorder>
  <zorder>Carta2B</zorder>
  <zorder>Carta3B</zorder>
  <zorder>Carta4B</zorder>
  <zorder>Carta5B</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
